src/js/providers/html5.js

```diff
@@ -501,7 +501,7 @@ define([
             }
 
             // if playlist item contains .vtt tracks, load them
-            if (utils.isIOS() && item) {
+            if (item) {
```

Comment:
Since we know we need this to happen when the source is set, let's get rid of this if statement, making sure an item is always passed to `_setVideotagSource`.